test/generators.ts

```diff
@@ -0,0 +1,67 @@
+import { object, nat, tuple, constant, array, anything } from 'fast-check'
+import { deepEquals } from '../src/deep-equals'
+import { entries } from '../src/entries';
+import { includes } from '../src/includes';
+import { values } from '../src/values';
+
+function shuffle<T>(array: Array<T>): Array<T> {
```

Comment:
const, not function, ReadonlyArray not Array

Reply:
Done